views/challenge.py

```diff
@@ -71,6 +88,10 @@ def has_challenge(self, key: ChallengeKey) -> bool:
     def get_challenge(self, key: ChallengeKey) -> Challenge:
         return self.challenges[key]
 
+    def get_challenge_cnt(self) -> int:
```

Comment:
maybe a property? Also using `count` is clearer than `cnt`

```
@property
def challenge_count
```

Reply:
Updated.